app/src/main/res/layout/activity_main.xml

```diff
@@ -265,4 +265,33 @@
             app:layout_constraintTop_toTopOf="@+id/lblGPSSpeed"
             app:layout_constraintVertical_bias="0.0" />
 
+        <Button
+            android:id="@+id/button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="512dp"
+            android:onClick="exportTest"
+            android:text="ExportTest"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <Button
+            android:id="@+id/button2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:onClick="schedulerTest"
+            android:text="ScheduleTest"
+            app:layout_constraintStart_toStartOf="@+id/button"
+            app:layout_constraintTop_toBottomOf="@+id/button" />
+
+        <Button
+            android:id="@+id/button3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:onClick="cancelJobs"
+            android:text="CancelTest"
+            app:layout_constraintStart_toStartOf="@+id/button2"
+            app:layout_constraintTop_toBottomOf="@+id/button2" />
```

Comment:
maybe already implement a switch instead of 2 buttons?

Reply:
these are here for testing purposes (new GUI will have a Switch)